.github/renovate.json

```diff
@@ -84,10 +85,19 @@
       "automerge": false,
       "enabled": true
     },
+    {
+      "matchDatasources": [
+        "rpm"
+      ],
+      "versioning": "regex:(?<major>\\d+)(\\.(?<minor>\\d+))?(\\.(?<patch>\\d+))?(-(?<prerelease>.*))?$",
+      "automerge": false,
```

Comment:
Maybe we should auto merge no ? We have it like that for iubuntu

Reply:
answered in the previous comment